build.sh

```diff
@@ -299,8 +300,6 @@ case "$COMMAND" in
 
     "osx-swift")
         xcrealmswift "-scheme 'RealmSwift OSX' -configuration $CONFIGURATION build"
-        rm -rf build/osx
```

Comment:
This does still need to be done to ensure that any files removed from the framework actually go away.